src/core/core_structs/simulation_sequence.jl

```diff
@@ -0,0 +1,28 @@
+mutable struct SimulationSequence
+    initial_time::Dates.DateTime
+    horizons::Dict{String, Int64}
+    intervals::Dict{String, <:Dates.TimePeriod}
+    feed_forward_chronologies::Dict{Pair{String, String}, <:AbstractChronology}
+    feedforward::Dict{Tuple{String, Symbol}, <:AbstractAffectFeedForward}
+    ini_cond_chronology::Dict{String, <:AbstractChronology}
```

Comment:
nits:  feedforward vs feed_forward.  Be consistent.
If the user is going to use ini_cond_chronology then spell out the whole thing.  If internal only, never mind.